resources/remote-control.txt

```diff
@@ -1,21 +1,38 @@
 Remote control protocol.
 
 Supported commands:
- f - Get frequency [Hz]
```

Comment:
I added description argument '?' to 'M' command. Without reformate the text is ugly.